Gemfile

```diff
@@ -26,5 +26,4 @@ end
 
 platform :rbx do
   gem 'json'
-  gem 'rubysl'
```

Comment:
[note] Removed `rubysl` to avoid the following error of `bundle install`:

```console
$ bundle install
Fetching gem metadata from https://rubygems.org/.......
Resolving dependencies...
Bundler found conflicting requirements for the Ruby version:
  In Gemfile:
    rainbow was resolved to 3.1.0, which depends on
      Ruby (>= 2.3.0)

    rubysl was resolved to 1.1.0, which depends on
      rubysl-irb (~> 1.0) was resolved to 1.0.2, which depends on
        Ruby (~> 1.8.7)
```

Reply:
I think we should drop the full :rbx platform group from the Gemfile. Any Rubinius-specific support: remove it.